stacks-signer/src/signerdb.rs

```diff
@@ -85,14 +93,30 @@ impl SignerDb {
 
     /// Insert a block into the database.
     /// `hash` is the `signer_signature_hash` of the block.
-    pub fn insert_block(&mut self, block_info: &BlockInfo) -> Result<(), DBError> {
+    pub fn insert_block(
+        &mut self,
+        reward_cycle: u64,
+        block_info: &BlockInfo,
+    ) -> Result<(), DBError> {
         let block_json =
             serde_json::to_string(&block_info).expect("Unable to serialize block info");
         let hash = &block_info.signer_signature_hash();
+        let block_id = &block_info.block.block_id();
+        let signed_over = &block_info.signed_over;
+        debug!(
+            "Inserting block_info: reward_cycle = {reward_cycle}, sighash = {hash}, block_id = {block_id}, signed = {signed_over} vote = {:?}",
+            block_info.vote.as_ref().map(|v| {
+                if v.rejected {
+                    "REJECT"
+                } else {
+                    "ACCEPT"
+                }
+            })
```

Comment:
Nit: I think it would be cleaner to have this logic as a `Display` implementation on the `BlockInfo` type. Any reason not to do that?

Reply:
This is a good idea. I'll add this in a separate PR.